### PR TITLE
Add duration to selected stack frame text.

### PR DESCRIPTION
**Pull request by kenzieschmoll**
Screen Shot 2019-05-01 at 10 17 22 AM

### CODE REVIEW

File: packages/devtools/lib/src/timeline/cpu_profile_protocol.dart [hunk 1]
```diff
@@ -161,8 +161,11 @@ class CpuStackFrame {
   }
 
   @override
-  String toString() {
-    return '$name ($sampleCount samples, '
-        '${percent2(cpuConsumptionRatio)})';
+  String toString({Duration duration}) {
+    String str = '$name ';
+    if (duration != null) {
+      str += '- ${msText(duration, fractionDigits: 2)} ';
```
Comment (jacob314) [inline packages/devtools/lib/src/timeline/cpu_profile_protocol.dart]: Add a TODO to use a number of fractionDigits that better reflects the actual resolution.
For example, for the screenshot given fractionDigits: 1 would be enough.
In other words, how long is a single sample? We not show precision more than 10X or maybe 100X the length of a single sample.

File: packages/devtools/lib/src/timeline/cpu_profile_protocol.dart [hunk 2]
```diff
@@ -161,8 +161,11 @@ class CpuStackFrame {
...
+    }
+    return str += '($sampleCount samples, ${percent2(cpuConsumptionRatio)})';
```
Comment (jacob314) [inline packages/devtools/lib/src/timeline/cpu_profile_protocol.dart]: nit: return the final concat string at the end
and use StringBuilder for clarity.

Comment (kenzieschmoll) [reply packages/devtools/lib/src/timeline/cpu_profile_protocol.dart]: done.

File: packages/devtools/lib/src/timeline/cpu_profile_protocol.dart (hunk 1 above)
Comment (kenzieschmoll) [reply packages/devtools/lib/src/timeline/cpu_profile_protocol.dart]: done.

**Pull request merged**